Packs/DuoAdminApi/Integrations/DuoEventCollector/DuoEventCollector.py

```diff
@@ -38,53 +38,117 @@ class Client:
     A class for the client request handling
     """
 
-    def __init__(self, params: Params):  # pragma: no cover type: ignore
-        self.params = params.get('params')  # type: ignore[attr-defined]
-        self.admin_api = create_api_call(params.get('host'),  # type: ignore[attr-defined]
-                                         params.get('integration_key'),  # type: ignore[attr-defined]
-                                         (params.get('secret_key')).get('password'))  # type: ignore[attr-defined]
+    def __init__(self, params: Params):
+        self.params = params
+        self.admin_api = create_api_call(self.params.host,
+                                         self.params.integration_key,
+                                         str(self.params.secret_key.get('password')))
 
-    def call(self, request_order: list) -> dict:  # pragma: no cover
+    def call(self, request_order: list) -> tuple:
```

Comment:
Since you're returning a tuple, please mention in the docstring what's expected to be in the tuple

Reply:
Done